day_two_guide/topics/ntp_synchronization.adoc

```diff
@@ -13,6 +13,15 @@ keeping and time stamps, and is highly recommended for Kubernetes, which
 election, health checks for pods and some other issues, and helps prevent time
 skew problems.
 
+[NOTE]
+====
+The {product-title} installation playbooks install, enable, and configure the
+`ntp` package to provide NTP service unless explicitly disabled by setting
```

Comment:
I'd split this sentence. Maybe "...NTP service by default. To disable this behavior, set.."